framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/InfoPanel/InfoPanel.module.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../styles/constants';
+
+.infoHeading {
+	font-family: Roboto;
+	font-style: normal;
+	font-weight: 300;
+	font-size: 14px;
+	line-height: 16.41px;
+	display: flex;
+	align-items: center;
+	height: 20px;
+	margin-bottom: 10px;
+
+	/* identical to box height */
+
+	/* Platinum gray */
+	color: #dfe6f2;
+}
+
+.infoContent {
```

Comment:
the `Text` component should be used so that it doesn't require the font settings

Reply:
The Text component does not support all the styles required here

Reply:
Actually, you are using {props.children}, so no need to apply the text style here. the user from the upper later will specify the style using `Text`